applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,397 @@
+.. _matter_adding_bridged_matter_device:
+
+Adding support for bridging of new Matter device type
+*****************************************************
+
+The Matter Bridge application supports bridging only a few Matter device types due to practical reasons.
```

Comment:
"bridging only a few" is correct here since the sentence ends "due to practical reasons". 

"only few" would work if the whole sentence was "Due to practical reasons, only few Matter device types are supported by the Matter Bridge application." and even then, "only a few" works better.

Using "bridging a few" would work without "due to practical reasons", in which case supporting a couple of Matter device types is something extra offered in the application.